content/en/armory-enterprise/quickspin/quickstart-quick-spin.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Quick Spin Quick Start 
+linkTitle: "Start using Quick Spin"
+no_list: true
+weight: 5
+description: >
+  Use this tutorial to lean how to create and manage a simple Spinnaker workflow.
+exclude_search: true
+---
+## Abstract
+This tutorial demonstrates how to:
```

Comment:
Instead of "Abstract", how about "Objectives" or "Learning Objectives" since that's what other  Armory tutorials use?

Reply:
We probably should create a standard heading to use since I am also writing tutorials in CDaaS.

http://localhost:1313/contribute/create/guides/content-guide/#tutorial